src/Authority/Authority.php

```diff
@@ -108,26 +108,38 @@ public function cannot($action, $resource, $resourceValue = null)
     /**
      * Define privilege for a given action and resource
      *
-     * @param string        $action Action for the rule
+     * @param array|string  $action Action for the rule
      * @param mixed         $resource Resource for the rule
      * @param Closure|null  $condition Optional condition for the rule
      * @return Rule
      */
     public function allow($action, $resource, $condition = null)
     {
+        if (is_array($action)) {
```

Comment:
Good catch, I will change that this afternoon and add some tests.

Concerning the return type, I think that the function should return what it does, a single rule if a single action is created and a collection if multiple are passed in.